src/scripts/main.js

```diff
@@ -1 +1,55 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const logo = body.querySelector('.logo');
+
+function createPromise(masegeClass, numberPromise, mouseEvent) {
```

Comment:
Consider moving the 'mouseEvent' parameter to the last position in the function declaration and provide a default value, such as 'click', to improve readability and avoid confusion with parameter order.